dataland-keycloak/dataland_theme/login/package.json

```diff
@@ -3,13 +3,18 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "build": "node build_styles.js"
+    "build": "node build_styles.js",
+    "updatepackages": "npm-check-updates -u",
+    "updatepackagelock": "npm update --save --dev && npm audit fix"
   },
   "dependencies": {
     "@ibm/plex": "^6.0.2",
-    "material-icons": "^1.11.5",
+    "material-icons": "^1.11.8",
     "primeflex": "^3.2.1",
     "primeicons": "^5.0.0",
-    "sass": "^1.53.0"
+    "sass": "^1.54.2"
```

Comment:
Kannst du noch einen Abschnitt in die readme schreiben, um diese package/package-lock zu updaten?

Reply:
Ich habe bereits in der Readme dazu vermerkt: "Do this in the frontend as well as in the keycloak theme".

Reply:
ok